frontend/src/app/shared/map/map.component.html

```diff
@@ -1,4 +1,6 @@
-<div id="map" style="height: 100%"></div>
```

Comment:
Why did you change that?
The map shouldn't have a fixed width or height initially.

Reply:
Added new map component